data/en/1_0102.json

```diff
@@ -82,47 +82,78 @@
                 }]
             },
             {
+                "questions": [{
+                    "description": "",
+                    "type": "General",
+                    "answers": [{
+                        "label": "",
+                        "description": "",
+                        "type": "Radio",
+                        "q_code": "d12",
+                        "id": "reporting-period-answer-1",
+                        "options": [{
+                                "label": "Yes",
+                                "value": "Yes"
+                            },
+                            {
+                                "label": "No",
+                                "value": "No"
+                            }
+                        ],
+                        "mandatory": true
+                    }],
+                    "id": "reporting-period-question-1",
+                    "title": "Are you able to report for the period from {{exercise.start_date|format_date}} to {{exercise.end_date|format_date}}?"
+                }],
+                "type": "Questionnaire",
+                "title": "Reporting period",
+                "id": "reporting-period-block-1",
```

Comment:
For the new id values I think we should have a clearer prefix of `reporting-period-choice` and drop the word `block`, so the id values would be:
```
reporting-period-choice
reporting-period-choice-question
reporting-period-choice-answer
```